packages/react/src/components/actionable-item/tests/ActionableItem.spec.tsx

```diff
@@ -61,6 +61,13 @@ describe('ActionableItem', () => {
                 expect(screen.getByText('hello world')).toBeVisible();
             });
 
+            it('should remove the active class from the children', () => {

```

Comment:
Can we change the test name to something like
`should not highlight dropdown options` or `should not apply CSS class to dropdown options`

Just to possibly make it more clear on what the test is trying to prove.